rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,133 @@
+
+1. Evaluate whether there is enough information in the ticket to inform the response. If not, write back to the user and ask for the missing information. The user has 2 weeks to respond or the ticket will be closed.
+
+2. Search the data portal documentation and FAQ for answers to the question. If the triage person can find an answer that resolves the problem, a response is sent to the user and the issue is closed. 
+
+3. If an answer is not evident from searching the Knowledge Base in the data portal, the triage person needs to ask for help from the secondary support team. The secondary support team is comprised of at least one member from each DCP component or service. Communication with the team is facilitated via the `zendesk` slack channel on [humancellatlas.slack.com](humancellatlas.slack.com). It may be obvious which team should receive the ticket; if so, then it can be assigned directly. If it’s not obvious, there can be some internal discussion with the secondary support staff about who should receive the ticket.
+
+4. At this point the ticket becomes the responsibility of the secondary support person, who should respond to the user and bring the ticket to an eventual close. The secondary support person MUST respond to the request within 2 days. 
+
+
+The triager will also work with the secondary support staff to continuously improve the DCP site and make common solutions findable and available to the end user. 
+
+1. They will tag each ticket with basic tags (data store/ingest/contribution/etc.) that help DCP staff categorize issues for later retrieval. In addition, Github tickets should be made for issues like: updating docs or writing new documentation and improving our FAQs, code examples, bug reports, etc. As much as possible, these solutions should be added to the Knowledge Base so that they can be turned into documentation or short FAQs and made available for users to access from the portal.
+
+2. They will create and maintain canned responses (how to connect/get involved with the HCA; how to submit data, etc.) that can be quickly sent to users who have written with those questions. These canned responses will be stored in the Knowledge Base.
+
+3. They will document discussions and decisions that arise from particular tickets in the Knowledge Base, so that it's clear who should respond to similar questions in the future and what the response should be.
+
+
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major component or service of the DCP. These representatives should have enough technical knowledge to answer most questions. They may require additional information from the person who created the ticket. They also may need to consult with technical experts who make up the tertiary support teams. Answers generated by the secondary support team will be added to the Knowledge Base by the secondary support team with the Triager's guidance.
+
```

Comment:
I keep running into cases where an expression is used like _Tertiary Support_ that has a forward reference which makes reading difficult.